picachu/modules/photos/queries/get_photos_query.py

```diff
@@ -45,11 +45,12 @@ def count_photos(gallery_id: int) -> int:
             current_session.close()
 
     @staticmethod
-    def by_limit() -> List:
+    def get_photos_list(gallery_id) -> List[Photo]:
         current_session = session()
         try:
             photos_list = current_session \
                 .query(Photo) \
+                .filter(Photo.gallery_id == gallery_id) \
                 .order_by(Photo.photo_file_path_s3.desc()) \
```

Comment:
In theory, sorting should be by date of download. The first photo in the list is the last one uploaded.